### PR TITLE
Convert :count when it is a Decimal number

### PR DESCRIPTION
**Pull request by jonhkr**
This is a fix for the `json` conversion when `:count` is a decimal number.

### CODE REVIEW

**Comment by jonhkr** [comment]
Ref: https://github.com/elixir-lang/ecto/pull/844

File: lib/phoenix_ecto/json.ex [hunk 1]
```diff
@@ -40,8 +40,12 @@ if Code.ensure_loaded?(Poison) do
 
     defp json_error(msg) when is_binary(msg), do: msg
     defp json_error({msg, count: count}) when is_binary(msg) do
-      String.replace(msg, "%{count}", Integer.to_string(count))
+      String.replace(msg, "%{count}", count_to_string(count))
```
Comment (josevalim) [inline lib/phoenix_ecto/json.ex]: Let's just call `to_string` which should convert everything properly.

**Comment by josevalim** [comment]
I will change it locally, thank you!

**Comment by jonhkr** [comment]
:+1:

**Pull request merged**